src/js/jquery_wrapper.js

```diff
@@ -0,0 +1,8 @@
+if ( typeof define === 'function' && define.amd && define.amd.jQuery ) {
+	define('$', (function (global) {
+		return function () {
+			var ret, fn;
+			return ret || global.$;
```

Comment:
This should probably be `global.jQuery`, to be in line with the `window.jQuery` below (support for noConflict).

Actually, why not `return ret || global.jQuery || global.Zepto`, removing the `&& define.amd.jQuery` check, and rename this file to `selector_library_wrapper.js`?